tests/integration/test_envelope.py

```diff
@@ -579,3 +579,51 @@ def test_sample_rates_metrics(mini_sentry, relay_with_processing, events_consume
 
     event, _ = events_consumer.get_event()
     assert event["_metrics"]["sample_rates"] == sample_rates
+
+
+# Checks that we buffer envelopes until we have a global config in processing mode.
```

Comment:
Could we move this comment as a docstring of the test?

Reply:
Why this resolved and not addressed?